scripts/code_coverage_report/generate_code_coverage_report/Sources/Utils/JSONParser.swift

```diff
@@ -18,15 +18,14 @@ import Foundation
 
 public enum JSONParser {
   // Decode an instance from a JSON file.
-  public static func readJSON<T: Codable>(of dataStruct: T.Type, from path: String) throws -> T? {
+  public static func readJSON<T: Codable>(of dataStruct: T.Type, from path: String) throws -> T {
     do {
       let fileURL = URL(fileURLWithPath: FileManager().currentDirectoryPath)
         .appendingPathComponent(path)
       let data = try Data(contentsOf: fileURL)
       return try JSONDecoder().decode(dataStruct, from: data)
     } catch {
-      print("\(path) cannot be read: \(error)\n")
+      fatalError("\(path) cannot be read: \(error)\n")
     }
```

Comment:
Nit: When the function itself throws, we don't need to wrap in a `do/catch`, we can just let the error propagate upwards.  This function can just be the contents of the `do` block.